configuration/pih/htmlforms/patientRegistration-contact.xml

```diff
@@ -27,6 +27,10 @@
 
                 setUpPhoneNumberRegex('<uimessage code="pihcore.invalid.phoneNumber"/>');
 
+                beforeSubmit.push(function() {
+                   return phoneNumberRegex(getValue('contactPhoneNumber.value'));
+                });
```

Comment:
These functions, and how they work, are defined here:

https://wiki.openmrs.org/display/docs/HTML+Form+Entry+JavaScript+Reference